### ISSUE TITLE
fix t2f mispellings

### ISSUE THREAD

**Issue by ewheeler** [unicef/etools-issues#243]
[correspondence](https://en.wiktionary.org/wiki/correspondence) is misspelled [here](https://github.com/unicef/etools-t2f/blob/staging/t2f/app-elements/api-elements/structure-data.html#L289)

[Clearance](https://en.wiktionary.org/wiki/clearance) is misspelled [here](https://github.com/unicef/etools-t2f/blob/staging/t2f/app-elements/api-elements/structure-data.html#L290)

**Comment by caktus-philip** [comment]
@ewheeler I can't find a misspelling of correspondence in that file. "Cleareance" is still incorrect. Any reason why these are related to migrations?

**Comment by ewheeler** [comment]
@caktus-philip hmm.
i recall being irked by `itenerary` in a table name somewhere.. so i had related all misspellings to the migration card just in case any of them necessitate a table name change (which could be easier to change while mucking with migrations)

**Comment by caktus-philip** [comment]
@ewheeler Yes, I remember the `itenerary`. It was fixed; I think Vinod squashed that one. Fortunately, fixing `Cleareance` in this file isn't related to migrations so it can be fixed at any time. I tried to push a PR for it but I don't have rights to that repo. I'll let someone else handle it. =)

**Comment by ewheeler** [comment]
ok! sounds good
